docs: Upgrade readme (#1060)
vtex/faststore: README.md

<p align="center">
  <a href="https://faststore.dev">
    <img alt="Faststore" src="./packages/ui/static/logo.png" width="60" />
  </a>
</p>
<h1 align="center">
  FastStore v1
</h1>
<p align="center">
  <strong>
    Fullstack ecommerce toolkit
  </strong>
</p>
<p align="center">
  FastStore is a fullstack toolkit based on React that helps developers build Performant, Stable, SEO and Analytics ready ecommerces in a matter of days.
</p>

- <strong>Performance</strong>: Keep yourself calm. We use strict budgets to ensure our libs are always light and fast.
- <strong>Stability</strong>: Generate your pages ahead by using a Jamstack ready toolkit.

> - <strong>SEO/Analytcs</strong>: Connect to your favorite analytics provider

Reviewer: ```suggestion
- <strong>SEO/Analytics</strong>: Connect to your favorite analytics provider.
```

- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce in your favorite React framework 
(Gatsby, Next.JS, Remix etc)

> - <strong>Speed</strong>: Customize one of our starters to your needs. Do not spend time creating tests, thinkig about architecture or infrastructure.

Reviewer: ```suggestion
- <strong>Speed</strong>: Customize one of our starters to your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.
```

## Getting started

> [Checkout the docs](https://faststore.dev/quickstart) to quickstart from one of our official starters or create your own.

Reviewer: ```suggestion
[Check out the docs](https://faststore.dev/quickstart) to quickstart from one of our official starters or create your own.
```

## Official Starters
- [Base](https://github.com/vtex-sites/base.store): Basic starter with generic store.

## Learn

> Check out our [Tutorials](https://faststore.dev/tutorials/overview), [How-to Guides](https://faststore.dev/how-to-guides/overview), [API References](https://faststore.dev/reference/overview) and [Conceptual guides](https://faststore.dev/conceptual-guides/overview) to deep dive in the world of building FastStores

Reviewer: ```suggestion
Check out our [Tutorials](https://faststore.dev/tutorials/overview), [How-to Guides](https://faststore.dev/how-to-guides/overview), [API References](https://faststore.dev/reference/overview) and [Conceptual guides](https://faststore.dev/conceptual-guides/overview) to deep dive in the world of building FastStores.
```

## Release Notes

> Follow our montly release notes at: https://faststore.dev/releases

Reviewer: ```suggestion
Follow our monthly release notes at: https://faststore.dev/releases
```

- [Marin Brasil](https://www.marinbrasil.com.br/)
- [Carrefour](https://mercado.carrefour.com.br/)

## Contributing
This is still a work in progress, however, if you are also an adventurous person, you can read the code and have some fun!

## Getting help

> If you find any issues on the project you would like to report, please create an [issue](https://github.com/vtex/faststore/issues) on the repository. If you have a question, idea or want to show us something cool you have built, feel free to create a [discussion](https://github.com/vtex/faststore/discussions).

Reviewer: ```suggestion
If you find any issues on the project you would like to report, please create an [issue](https://github.com/vtex/faststore/issues) on the repository. If you have a question, idea, or want to show us something cool you have built, feel free to create a [discussion](https://github.com/vtex/faststore/discussions).
```

## Packages

| Package                      | Description                                                                         |                                                                  Status                                                                   |
| :--------------------------- | :---------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------: |

> | @faststore/ui               | Exports basic Store components. Checkout our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |

Reviewer: ```suggestion
| @faststore/ui               | Exports basic Store components. Check out our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |
```

> | @faststore/ui               | Exports basic Store components. Check out our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |

Reviewer 2: Noticed this duplicate badge for tree shaking:

```suggestion
| @faststore/ui               | Exports basic Store components. Check out our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |
```

> | @faststore/sdk               | Exports basic logic hooks for creating your custom ecommerce                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk)  

Reviewer 2: Minor typo:

```suggestion
| @faststore/sdk               | Exports basic logic hooks for creating your custom ecommerce                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk)  
```

> | @faststore/api               | Connect your site to your ecommerce provider                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://badge.fury.io/js/%40faststore%2Fapi) [![npm version](https://badgen.net/bundlephobia/dependency-count/@faststore/api)](https://bundlephobia.com/package/@faststore/api)     

Reviewer 2: Also a duplicate NPM version badge:

```suggestion
| @faststore/api               | Connect your site to your ecommerce provider                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://bundlephobia.com/package/@faststore/api)     
```
